### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [rajibdpi/Jekyll-bootstrap4#1]
The user @carlsonsantana validate your site template "https://rajibahmed.me/Jekyll-bootstrap4/" and found these accessibility errors:
* Page https://rajibahmed.me/Jekyll-bootstrap4/:
  * Issue *747285ef-3c8d-4097-9071-26a18b0833e5*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation: change text colour to #0072ed.
    * **Selector**: `html > body > footer > div > div:nth-child(2) > ul > li:nth-child(2) > a > span:nth-child(2)`
    * **Context**: `<span class="username">jekyllrb</span>`
  * Issue *8dcea3cb-db86-450f-b30f-69ed336da529*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation: change text colour to #0072ed.
    * **Selector**: `html > body > footer > div > div:nth-child(2) > ul > li:nth-child(1) > a > span:nth-child(2)`
    * **Context**: `<span class="username">jekyll</span>`
  * Issue *3763659c-afab-4004-9220-5eb4be86594b*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation: change text colour to #0072ed.
    * **Selector**: `html > body > footer > div > div:nth-child(1) > ul > li:nth-child(2) > a`
    * **Context**: `<a href="mailto:your-email@example.com">your-email@example.com</a>`
  * Issue *43706345-bb78-4632-9d95-d20a68dc7d98*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.98:1. Recommendation: change background to #0072ed.
    * **Selector**: `html > body > main > div:nth-child(1) > div > p:nth-child(3) > a`
    * **Context**: `<a class="btn btn-primary btn-lg" href="https://github.com/rajibdpi/Jekyll-bootstrap4/archive/v2.3.4.zip" role="button">Download</a>`
  * Issue *2bf5980a-019d-40be-930a-1d34fe1d1ede*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
    * **Message**: This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.67:1. Recommendation: change background to #262c32.
    * **Selector**: `#navbarsExampleDefault > form > button`
    * **Context**: `<button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/10d8bf9d-e963-4d68-bcdd-c752dbebfe3f/).
